homeassistant/components/fastdotcom/__init__.py

```diff
@@ -40,17 +36,17 @@
 )
 
 
-async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+async def async_setup_platform(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
This should only now import to the config flow. This still sets up the integration

Reply:
Seems the import should be from the sensor so essentially this could be deleted and remaining should go into `setup_entry`

Reply:
Effectively meaning the entire async_setup_platform can be deleted in __init__.py?

Reply:
Not yet as people can have a yaml config so it needs to be kept but can simply return true until the yaml config can be removed completely.

Reply:
Clear, thanks. Code has been changed! :)